specs.json

```diff
@@ -179,7 +179,12 @@
   },
   {
     "url": "https://www.w3.org/TR/css-cascade-4/",
-    "shortTitle": "CSS Cascading 4"
+    "shortTitle": "CSS Cascading 4",
+    "forceCurrent": true
+  },
+  {
```

Comment:
Linter is picky and wants entries to be sorted by (otherwise opaque :)) URLs, so this entry needs to go near the top...